coordinatorlib/src/main/scala/com/socrata/datacoordinator/secondary/PlaybackToSecondary.scala

```diff
@@ -265,6 +265,26 @@ class PlaybackToSecondary[CT, CV](u: PlaybackToSecondary.SuperUniverse[CT, CV],
       retrying[T](actions, filter)
     }
 
+    /**
+      * Serialize resync in the same store group
+      * because resync will cause reads to be unavailable (at least in pg)
+      */
+    def resyncSerially(): Unit = {
+      val sm = u.secondaryManifest
+      try {
+        // resync immediately calls commit before actual work is done.
+        // That allows us to see resync in progress from the resync table.
+        if (sm.lockResync(datasetId, secondary.storeId, secondary.groupName) > 0) {
+          resync()
```

Comment:
insert should always return 1 here if it is successful.  If someone else is in resync, a sql exception will happen.  It is either a duplicated key or timeout (after 3s) depending on whether the other process has committed the resync entry.  If it is a sql exception, it should trigger the special re-enqueue.  If it is something else, it should trigger a generic re-enqueue.